### ISSUE TITLE
[Codacy code cleanup] occurrence: 1688 pattern: Double quote to prevent globbing and word splitting

### ISSUE THREAD

**Issue by rkothiya** [Seagate/cortx-motr#1606]
### Problem

We see 1688 occurrence of pattern, "Double quote to prevent globbing and word splitting". 

### Expected behavior

We should not be getting these warnings 

### How to reproduce

Fire the below command to list all the warnings : 

`# shellcheck   -x  <file name>`

### Deployment information

Install the shellcheck package. For example to install it on Fedora/RHEL/centos you can fire : 

` # yum -y install ShellCheck`

### Additional information

Following files contains this warnings 

./addb2/st/addb2dump-plugin.sh
./cas/st/ctgdump_tesh.sh
./console/st/console-st.sh
./dix/cm/st/m0t1fs_dix_repair.sh
./dix/cm/st/m0t1fs_dix_repair_quiesce.sh
./fdmi/plugins/fdmi_plugin_st.sh
./fdmi/st/echo/echo_plugin_start.sh
./fdmi/st/fdmi_test.sh
./m0t1fs/linux_kernel/st/m0t1fs_cc_io_with_sns_repair.sh
./m0t1fs/linux_kernel/st/m0t1fs_client.sh
./m0t1fs/linux_kernel/st/m0t1fs_client_inc.sh
./m0t1fs/linux_kernel/st/m0t1fs_common_inc.sh
./m0t1fs/linux_kernel/st/m0t1fs_dgmode_io.sh
./m0t1fs/linux_kernel/st/m0t1fs_failure_after_sns_repair_quiesce.sh
./m0t1fs/linux_kernel/st/m0t1fs_fsync_test.sh
./m0t1fs/linux_kernel/st/m0t1fs_fwait_test.sh
./m0t1fs/linux_kernel/st/m0t1fs_multi_clients.sh
./m0t1fs/linux_kernel/st/m0t1fs_pool_version_assignment.sh
./m0t1fs/linux_kernel/st/m0t1fs_poolmach.sh
./m0t1fs/linux_kernel/st/m0t1fs_rconfc_fail_test.sh
./m0t1fs/linux_kernel/st/m0t1fs_restart_motr_service.sh
./m0t1fs/linux_kernel/st/m0t1fs_rlock_revoke.sh
./m0t1fs/linux_kernel/st/m0t1fs_rpc_cancel_test.sh
./m0t1fs/linux_kernel/st/m0t1fs_server.sh
./m0t1fs/linux_kernel/st/m0t1fs_server_inc.sh
./m0t1fs/linux_kernel/st/m0t1fs_sns_common_inc.sh
./m0t1fs/linux_kernel/st/m0t1fs_sns_repair_1f.sh
./m0t1fs/linux_kernel/st/m0t1fs_sns_repair_1k_1f.sh
./m0t1fs/linux_kernel/st/m0t1fs_sns_repair_1n_1f.sh
./m0t1fs/linux_kernel/st/m0t1fs_sns_repair_abort.sh
./m0t1fs/linux_kernel/st/m0t1fs_sns_repair_abort_quiesce.sh
./m0t1fs/linux_kernel/st/m0t1fs_sns_repair_ios_fail.sh
./m0t1fs/linux_kernel/st/m0t1fs_sns_repair_mf.sh
./m0t1fs/linux_kernel/st/m0t1fs_sns_repair_quiesce.sh
./m0t1fs/linux_kernel/st/m0t1fs_sns_repair_shutdown.sh
./m0t1fs/linux_kernel/st/m0t1fs_spare_space_reserve.sh
./m0t1fs/linux_kernel/st/m0t1fs_test.sh
./motr/st/utils/ha_msg_send.sh
./motr/st/utils/m0crate_iem_freespace_st.sh
./motr/st/utils/m0crate_st.sh
./motr/st/utils/m0kv_start.sh
./motr/st/utils/motr_client_st.sh
./motr/st/utils/motr_cmd.sh
./motr/st/utils/motr_conf_update_st.sh
./motr/st/utils/motr_device_util.sh
./motr/st/utils/motr_dgmode_io_st.sh
./motr/st/utils/motr_fs.sh
./motr/st/utils/motr_local_conf.sh
./motr/st/utils/motr_raid0_st.sh
./motr/st/utils/motr_remote_conf.sh
./motr/st/utils/motr_rm_lock_cc_io.sh
./motr/st/utils/motr_rmw_st.sh
./motr/st/utils/motr_rpc_cancel_test.sh
./motr/st/utils/motr_services.sh
./motr/st/utils/motr_sns_common.sh
./motr/st/utils/motr_sns_repair_N_1.sh
./motr/st/utils/motr_sns_repair_st.sh
./motr/st/utils/motr_st_inc.sh
./motr/st/utils/motr_sync_replication_st.sh
./motr/st/utils/motr_sys_test.sh
./motr/st/utils/motr_utils_st.sh
./motr/st/utils/sns_failure_after_repair_quiesce.sh
./motr/st/utils/sns_repair_common_inc.sh
./motr/st/utils/sns_repair_motr_1f.sh
./motr/st/utils/sns_repair_motr_1k_1f.sh
./motr/st/utils/sns_repair_motr_1n_1f.sh
./motr/st/utils/sns_repair_motr_abort.sh
./motr/st/utils/sns_repair_motr_abort_quiesce.sh
./motr/st/utils/sns_repair_motr_ios_fail.sh
./motr/st/utils/sns_repair_motr_mf.sh
./motr/st/utils/sns_repair_motr_quiesce.sh
./motr/st/utils/sns_repair_motr_shutdown.sh
./motr/st/utils/spiel_sns_motr_repair.sh
./motr/st/utils/spiel_sns_motr_repair_quiesce.sh
./net/lnet/st/linux_kernel/m0lnetping.sh
./net/test/demo/demo-list-nids.sh
./net/test/demo/demo-test-run.sh
./net/test/demo/demo.sh
./net/test/st/run-1x1.sh
./net/test/st/st-bulk.sh
./net/test/st/st-ping.sh
./net/test/test.sh
./net/test/demo/test.sh
./rpc/it/linux_kernel/rpcload.sh
./rpc/ub/plot.sh
./scripts/addb-py/chronometry/fom_req.sh
./scripts/addb-py/chronometry/restart_ios.sh
./scripts/addb-py/chronometry/s3_build_fix/rebuildall.sh
./scripts/build-prep-1node-cortx-mgw.sh
./scripts/build-prep-1node.sh
./scripts/demo/2015-10-dev2.sh
./scripts/install/opt/seagate/cortx/motr/common/cortx_util_funcs.sh
./scripts/install/opt/seagate/cortx/motr/common/m0_sns_utils_common.sh
./scripts/install/opt/seagate/cortx/motr/libexec/m0addb_logrotate.sh
./scripts/install/opt/seagate/cortx/motr/libexec/m0trace_logrotate.sh
./scripts/install/opt/seagate/cortx/motr/libexec/motr_cfg.sh
./scripts/install/opt/seagate/cortx/motr/libexec/motr_data_recovery.sh
./scripts/install/opt/seagate/cortx/motr/sanity/cortx_lnet_sanity.sh
./scripts/install/opt/seagate/cortx/motr/sanity/cortx_srvc_sanity.sh
./scripts/install/opt/seagate/cortx/motr/sanity/motr_sanity.sh
./scripts/provisioning/vmhost/create-cluster.sh
./scripts/provisioning/vmhost/destroy-cluster.sh
./scripts/provisioning/vmhost/gci-compile-install-hare.sh
./scripts/provisioning/vmhost/gci-create-cluster.sh
./scripts/provisioning/vmhost/prepare-host.sh
./scripts/provisioning/vmhost/super-cleanup.sh
./scripts/systemtap/kem/kem_run.sh
./scripts/systemtap/stp-runner.sh
./sns/cm/st/repair_test.sh
./spiel/st/m0t1fs_spiel_dix_common_inc.sh
./spiel/st/m0t1fs_spiel_dix_repair.sh
./spiel/st/m0t1fs_spiel_dix_repair_quiesce.sh
./spiel/st/m0t1fs_spiel_sns_common_inc.sh
./spiel/st/m0t1fs_spiel_sns_repair.sh
./spiel/st/m0t1fs_spiel_sns_repair_quiesce.sh
./spiel/st/m0t1fs_spiel_st.sh
./spiel/st/spiel_multiple_confd.sh
./st/m0d-device-detach-test.sh
./st/m0d-fsync-test.sh
./st/m0d-signal-test.sh
./st/m0mt-singlenode.sh
./st/m0t1fs-buffsize-layout-change-test.sh
./st/sanity.sh
./st/stob_domain_recreate_on_corruption.sh

**Comment by rkothiya** [comment]
The below pull request addresses many warnings but not all. Since the number of occurrence of these warnings are huge I will resolve these in multiple patches which should make the review easier.
https://github.com/Seagate/cortx-motr/pull/1621

**Comment by cortx-admin** [comment]
> Rinku Kothiya commented in Jira Server:

The below pull request addresses many warnings but not all. Since the number of occurrence of these warnings are huge I will resolve these in multiple patches which should make the review easier.
[https://github.com/Seagate/cortx-motr/pull/1621]

**Comment by cortx-admin** [comment]
> Rinku Kothiya commented in Jira Server:

I have fired the below job to test this patch : 
[https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1135/]

**Comment by cortx-admin** [comment]
> Rinku Kothiya commented in Jira Server:

Screen shot of the test results have been attached to this ticket.

*The below 3 tests failed :* 
    00userspace-tests
    52motr-singlenode-sanity
    44motr-rm-lock-cc-io  <<<<<<<<<<<<< Only failure which is suspected due to proposed patch.

I have attached the logs of all the 3 failures to this ticket for details.

**Comment by cortx-admin** [comment]
> Rinku Kothiya commented in Jira Server:

Updated the pull request and fired the test again : 
[https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1137/]

**Comment by cortx-admin** [comment]
> Rinku Kothiya commented in Jira Server:

This time also we had 44motr-rm-lock-cc-io failed.

*Failure not related to the patch :*
{code:java}
motr[00000]:  6fa0  ERROR  [st/utils/helper.c:178:client_init]  <! rc=-22
config parameters not initialized.
Client init failed: rc=-22
pver is : <0:0>
motr[00000]:  5dc0  ERROR  [st/utils/helper.c:178:client_init]  <! rc=-22
config parameters not initialized.
init failed! rc = -22
81.82user 10.73system 3:10.88elapsed 48%CPU (0avgtext+0avgdata 555448maxresident)k
2096inputs+2782792outputs (42major+976525minor)pagefaults 0swaps{code}
Attaching the error log with the run number :
 Run-1137-44motr-rm-lock-cc-io.stderr.log

*Previous failure was related to the patch :*
{code:java}
----- run_st 44motr-rm-lock-cc-io -----
<< 44motr-rm-lock-cc-io >>
/root/motr/motr_test_github_workdir/workdir/src/motr/st/utils/../../../m0t1fs/linux_kernel/st/m0t1fs_client_inc.sh: line 749: unexpected EOF while looking for matching `''
/root/motr/motr_test_github_workdir/workdir/src/motr/st/utils/../../../m0t1fs/linux_kernel/st/m0t1fs_client_inc.sh: line 915: syntax error: unexpected end of file
{code}

**Comment by rkothiya** [comment]
Another pull request is : 
https://github.com/Seagate/cortx-motr/pull/1644

**Comment by cortx-admin** [comment]
> Rinku Kothiya commented in Jira Server:

For the second pull request i.e. ([https://github.com/Seagate/cortx-motr/pull/1644]) I have fired the below test : 
[https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1142/]

**Comment by rkothiya** [comment]
Test results : 
<img width="533" alt="image" src="https://user-images.githubusercontent.com/44196748/165231041-e3128bf7-304c-4224-8075-a523a54e465f.png">

**Comment by rkothiya** [comment]
I see there were 15 failures, but it does not seem related to this patch. Another pair of eyes on the test results would be preferred to confirm the same.

**Comment by cortx-admin** [comment]
> Rinku Kothiya commented in Jira Server:

!image-2022-04-26-11-20-38-190.png!

**Comment by cortx-admin** [comment]
> Rinku Kothiya commented in Jira Server:

The test results without the patch show the below 2 failures : 


{code:java}
01motr-single-node.52motr-singlenode-sanity.test
01motr-single-node.00userspace-tests.test{code}

But with the pr 1621 there is 1 extra failure while with the pr 1644 there are 13 extra failure.  
We need to fix that and update the pr before we can merge it.

**Comment by johnbent** [comment]
@rkothiya and @alfhad , perhaps the simplest solution is to just remove those small number of changes? So, if Alfhad fixed the pattern in 500 places and it is breaking in 7 places, just remove the fix in those 7 places. Then, at least we can still land the 493 fixes and defer the remaining 7 for the future.

**Comment by rkothiya** [comment]
@johnbent , sure, that is what we have planned. Also for the first patch https://github.com/Seagate/cortx-motr/pull/1621 I already achieved this. Below are the test results which shows the same number of failures with and without the patch so we should be able to land pr #1621 

<img width="557" alt="Test REsult 1157" src="https://user-images.githubusercontent.com/44196748/166136405-14e020d4-07e3-446b-81bf-14d95f39e6a6.png">

**Comment by johnbent** [comment]
Awesome. Very happy to hear it. Thanks again for mentoring this community person. Great work.
________________________________

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: Rinku Kothiya
* Component Build  :  5102 
* Release Build    :  5384  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5384 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5384
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5384
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5384
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5384 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: fahadshah2411
* Component Build  :  5113 
* Release Build    :  5412  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5412 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5412
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5412
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5412
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5412 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: fahadshah2411
* Component Build  :  5124 
* Release Build    :  5433  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5433 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5433
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5433
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5433
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5433 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: fahadshah2411
* Component Build  :  5125 
* Release Build    :  5434  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5434 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5434
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5434
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5434
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5434 
{panel}

**Comment by cortx-admin** [comment]
> Chandradhar Raval commented in Jira Server:

Hi [~934465],

I see multiple PRs were raised for CORTX-30751(#1732, #1733, #1736, #1737, #1738, #1739)? Are they all fixing one pattern mentioned in the summary here?

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: fahadshah2411
* Component Build  :  5129 
* Release Build    :  5445  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5445 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5445
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5445
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5445
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5445 
{panel}

**Comment by chandradharraval** [comment]
Hi @rkothiya ,
Any update on above comments?

**Comment by cortx-admin** [comment]
> Rinku Kothiya commented in Jira Server:

Yes these all are fixing one pattern mentioned in the summary here (#1732, #1733, #1736, #1737, #1738, #1739).

Because there are too many occurrence's of these patterns hence we have multiple patches sent to address this issue.

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: Rinku Kothiya
* Component Build  :  5194 
* Release Build    :  5665  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5665 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5665
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5665
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5665
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5665 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: fahadshah2411
* Component Build  :  5207 
* Release Build    :  5700  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5700 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5700
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5700
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5700
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5700 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: fahadshah2411
* Component Build  :  5211 
* Release Build    :  5707  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5707 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5707
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5707
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5707
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5707 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: fahadshah2411
* Component Build  :  5212 
* Release Build    :  5708  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5708 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5708
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5708
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5708
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5708 
{panel}

**Comment by rkothiya** [comment]
added pr https://github.com/Seagate/cortx-motr/pull/1956 to partially address this.

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: 89624145+zohebkhann
* Component Build  :  5258 
* Release Build    :  5818  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5818 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5818
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5818
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5818
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5818 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: 89624145+zohebkhann
* Component Build  :  5277 
* Release Build    :  5865  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5865 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5865
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5865
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5865
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5865 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: 89624145+zohebkhann
* Component Build  :  5281 
* Release Build    :  5876  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5876 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5876
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5876
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5876
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5876 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: 89624145+zohebkhann
* Component Build  :  5296 
* Release Build    :  5905  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5905 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5905
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5905
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5905
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5905 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: fahadshah2411
* Component Build  :  5306 
* Release Build    :  5923  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5923 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5923
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5923
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5923
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5923 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: gaurav.gaur
* Component Build  :  5307 
* Release Build    :  5924  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5924 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5924
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5924
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5924
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5924 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: fahadshah2411
* Component Build  :  5309 
* Release Build    :  5930  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5930 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5930
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5930
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5930
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5930 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: Rinku Kothiya
* Component Build  :  5310 
* Release Build    :  5931  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5931 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5931
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5931
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5931
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5931 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: Rinku Kothiya
* Component Build  :  5311 
* Release Build    :  5933  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5933 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5933
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5933
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5933
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5933 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: Rinku Kothiya
* Component Build  :  5313 
* Release Build    :  5936  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5936 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5936
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5936
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5936
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5936 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: Rinku Kothiya
* Component Build  :  5314 
* Release Build    :  5937  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5937 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5937
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5937
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5937
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5937 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: Rinku Kothiya
* Component Build  :  5315 
* Release Build    :  5939  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5939 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5939
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5939
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5939
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5939 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: fahadshah2411
* Component Build  :  5318 
* Release Build    :  5954  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5954 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5954
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5954
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5954
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5954 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: fahadshah2411
* Component Build  :  5325 
* Release Build    :  5969  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5969 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5969
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5969
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5969
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5969 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: fahadshah2411
* Component Build  :  5326 
* Release Build    :  5970  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5970 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5970
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5970
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5970
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5970 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: Rinku Kothiya
* Component Build  :  5333 
* Release Build    :  5978  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/5978 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5978
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5978
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-5978
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-5978 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: fahadshah2411
* Component Build  :  5358 
* Release Build    :  6029  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/6029 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-6029
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-6029
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-6029
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-6029 
{panel}

**Comment by cortx-admin** [comment]
> Gaurav Chaudhari commented in Jira Server:

{panel:bgColor=#c1c7d0}h2. motr - main branch build pipeline SUCCESS
h3. Build Info:  
* Author: fahadshah2411
* Component Build  :  5360 
* Release Build    :  6033  

  h3. Artifact Location  :  
*  http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/6033 

h3. Image Location  :  
*  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-6033
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-6033
cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-6033
cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-6033 
{panel}

**Comment by shailesh-vaidya** [comment]
Closing as an obsolete
